vulture_os/toolkit/api_parser/api_parser.py

```diff
@@ -83,7 +84,7 @@ def __init__(self, data):
             else:
                 self.proxies = self.get_system_proxy()
 
-        self.redis_cli = RedisBase()
+        self.redis_cli = RedisBase(password=Cluster.get_global_config().redis_password)
```

Comment:
Config object is loaded above, would be a shame not to use it...

```suggestion
        self.redis_cli = RedisBase(password=config.redis_password)
```

Reply:
Done